components/index.ts

```diff
@@ -0,0 +1,47 @@
+export { Flex, Box, Link } from 'rebass/styled-components'
+
+import colors from './theme/colors'
+import theme from './theme/rebassTheme'
+
+export {
+  Button,
+  Heading,
+  Text,
+  Input,
+  IconButton,
+  RadioButton,
+  RadioGroup,
+  InputWithIconButton,
+  HeroLead,
+  Hero,
+  LogoOONIRun,
+  TwitterShareButton,
+  FacebookShareButton,
+  Modal,
+  Container,
+  Image,
+} from './components'
+
+export {
+  Label,
+  Select,
+  Textarea,
+  Radio,
+  Checkbox,
+  Slider,
+  Switch,
+} from '@rebass/forms/styled-components'
+
+// export * from './icons'
```

Comment:
Does this break any of the existing import paths?

Can we make the import part a bit cleaner, like `ooni-components/icons` instead of having to put `dist` in the middle?

Reply:
Currently in `ooni/explorer`, the import path of icons is like this only -> 'ooni-components/dist/icons'.
I kept it like this to avoid any breaking 😅